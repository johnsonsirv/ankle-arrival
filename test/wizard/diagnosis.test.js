/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import DiagnosisPage from '../../src/wizard/diagnosis';

function setup() {
  const props = {
    diagnosis: {
      id: 18,
      injury: 'abdominal',
      disease: 'blunt abdominal injury',
      symptoms: 'excessive abdominal',
      player: 'Joe Smith',
      inference:
        'if player exhibits symptoms excessive abdominal, in abdominal affected area, then there is a blunt abdominal injury disease.',
      treatment: 'surgery',
      lifestyle: 'maintain healthy lifestyle',
    },
  };

  const enzymeWrapper = shallow(<DiagnosisPage {...props} />);
  return { props, enzymeWrapper };
}

describe('<DiagnosisPage /> rendering', () => {
  const { enzymeWrapper } = setup();
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
