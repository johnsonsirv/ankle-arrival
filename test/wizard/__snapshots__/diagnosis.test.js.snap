// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DiagnosisPage /> rendering renders correctly 1`] = `ShallowWrapper {}`;
